addons/account/models/account_invoice.py

```diff
@@ -1164,6 +1171,18 @@ class AccountInvoiceTax(models.Model):
     sequence = fields.Integer(help="Gives the sequence order when displaying a list of invoice tax.")
     company_id = fields.Many2one('res.company', string='Company', related='account_id.company_id', store=True, readonly=True)
     currency_id = fields.Many2one('res.currency', related='invoice_id.currency_id', store=True, readonly=True)
+    amount_in_group = fields.Monetary(compute='_compute_amount_in_group')
```

Comment:
no comprendo...

it seems you implemented a super easy thing in a super complex way. Do as csn propose, it's way more clean